src/pages/library/DataBrowserDetails.js

```diff
@@ -0,0 +1,82 @@
+import { Fragment, useState } from 'react'
+import { div, h, h1 } from 'react-hyperscript-helpers'
+import FooterWrapper from 'src/components/FooterWrapper'
+import { centeredSpinner, icon } from 'src/components/icons'
+import { libraryTopMatter } from 'src/components/library-common'
+import * as Utils from 'src/libs/utils'
+
+
+const activeTab = 'browse & explore'
+const styles = {
+  page: { padding: 20, marginTop: 15 }
```

Comment:
TOL: Would this be more accurate as "content"

Reply:
sure, i'll push up this change